src/initial-submission/components/EditorsForm.tsx

```diff
@@ -161,6 +177,37 @@ const EditorsForm = ({ initialValues, ButtonComponent }: StepProps): JSX.Element
                 selectedPeople={suggestedReviewingEditors}
                 className="reviewing-editors-picker"
             />
+            <ExcludedToggle
+                togglePrefixText={t('editors.opposed-reviewing-editors-toggle-prefix')}
+                toggleActionText={t('editors.opposed-reviewing-editors-toggle-action-text')}
+                onClose={closeOpposedReviewers}
+            >
+                <PeoplePicker
+                    label={t('editors.reviewers-people-picker-label')}
+                    people={
+                        loadingReviewingEditors
+                            ? []
+                            : getReviewingEditors.getEditors.filter(ed => !suggestedReviewingEditors.includes(ed.id))
+                    }
+                    onRemove={(selected): void =>
+                        setValue(
+                            'opposedReviewingEditors',
+                            opposedReviewingEditors.filter(personId => personId !== selected),
+                        )
+                    }
+                    setSelectedPeople={(selected): void => setValue('opposedReviewingEditors', selected)}
+                    selectedPeople={opposedReviewingEditors}
+                    className="opposed-reviewing-editors-picker"
+                />
+                <MultilineTextField
+                    id="opposedReviewersReason"
+                    register={register}
+                    labelText={t('editors.opposed-reviewering-editor-reason-label')}
+                    invalid={errors && errors.opposedReviewersReason !== undefined}
+                    helperText={errors && errors.opposedReviewersReason ? errors.opposedReviewersReason.message : null}
+                />
+            </ExcludedToggle>
+            {/*TODO: translationforprefix*/}
```

Comment:
Can we put in this translation?

Reply:
Which part?

Reply:
I believe it's already translated. Th is is a left over todo from the merge. I'll remove it